lib/fika/parser/types.ex

```diff
@@ -105,13 +104,19 @@ defmodule Fika.Parser.Types do
     |> label("nothing")
     |> reduce({Helper, :to_atom, []})
 
+  bool_type =
+    string("Bool")
+    |> label("boolean")
+    |> reduce({Helper, :to_atom, []})
+
```

Comment:
Can we also add tests under `describe "types"` in test/fika/parser_test.exs for bool and map types?